vendor/engines/projects/app/services/projects/global_search/project_searcher.rb

```diff
@@ -0,0 +1,37 @@
+module Projects
+
+  module GlobalSearch
+
+    class ProjectSearcher
+
+      include ::GlobalSearch::Common
+
+      def template
+        "projects"
+      end
+
+      private
+
+      def query_object
+        if facility.try(:single_facility?)
+          facility.projects
+        else
+          Projects::Project
+        end
+      end
+
+      def execute_search_query
+        query_object.where("lower(name) = ?", query).select do |project|
```

Comment:
I agree it should be case-insensitive, but it should probably support partial matches as well.